sensor_msgs/PointCloud2 cloud
---
FeatureVector surface_features